kymatio/scattering1d/frontend/torch_frontend.py

```diff
@@ -67,36 +67,26 @@ def load_filters(self):
                     n += 1
 
     def scattering(self, x):
+        self.load_filters()
```

Comment:
why called before checks?

Reply:
good question! because the goal is for this method to become:

```
def scattering(self, x):
    self.load_filters()
    super(ScatteringBase1D, self).scattering(x)
```